alf/utils/common.py

```diff
@@ -95,6 +97,31 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def allow_child_to_ptrace(child_pid: int) -> None:
+    """In the *parent* process: allow a given child to ptrace us.
```

Comment:
Add explanation why we need to allow child to ptrace us. That is, avoiding "pidfd_getfd: Operation not permitted" for Tensor sharing.